commands/orchestrationcommands/stackresourcecommands/commands.go

```diff
@@ -0,0 +1,17 @@
+package stackresourcecommands
+
+import "github.com/rackspace/rack/internal/github.com/codegangsta/cli"
+
+var commandPrefix = "orchestration stack-resource"
```

Comment:
I keep going back and forth on this. Are there any other types of resources other than stack resources or any other types of events other than stack events (for example, is a template resource a thing?) ? If not, I think I like what you had before where it's just `resource` and `event`.